reference/test/matrix/fft_kernels.cpp

```diff
@@ -39,9 +39,15 @@ class Fft : public ::testing::Test {
     Fft()
         : exec(gko::ReferenceExecutor::create()),
           rng{7381},
+#ifdef GINKGO_FAST_TESTS
+          n1{2},
```

Comment:
I think to test the complex arithmetic, it would be best if each dimension is at least 4